docs/grid/your_first_grid.md

```diff
@@ -40,57 +40,6 @@ $ bin/console make:grid
 
 Now we can configure our first grid:
 
-{% tabs %}
```

Comment:
Are we sure it's not too early to entirely remove these two tabs ? Couldn't we just mark them as "YAML (Legacy)" and "PHP (Legacy)" ?

Reply:
Cannot we just put php service grid in the first place? Yaml/Php configs aren't event officially deprecated yet